.github/workflows/upload-appimage.yml

```diff
@@ -52,18 +40,19 @@ jobs:
           sudo apt-get install gh
 
           
-      - name: Delete existing comments
+      - name: Delete existing bot comments
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PR_NUMBER: ${{ github.event.number }}
         run: |
-          # Get all comments on the PR
-          COMMENTS=$(gh api repos/${{ github.repository }}/issues/$PR_NUMBER/comments --jq '.[].id')
-
-          # Loop through each comment and delete it
+          # Get all comments on the PR where the author is the bot
```

Comment:
suggestion: There is also `gh pr comment --edit-last`. Maybe we can try this flow so we try to edit first, if that fails we create a new comment.

gh pr comment --edit-last || gh pr comment

Reply:
Yes, this can be tried as well. It will avoid the extra calls to the github APIs.
```
      --edit-last        Edit the last comment of the same author
```

Reply:
ok, I will try to put conditional statement. Will try first editing the comment.